### PR TITLE
1.9.0

### PR DESCRIPTION
**Pull request by dandavison**

### CODE REVIEW

**Comment by Sushisource** [comment]
I might request we sneak https://github.com/temporalio/sdk-core/pull/858 in. I believe Core is up-to-date enough that no other changes should be required besides bumping once that's merged.

File: .github/workflows/build-binaries.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - "releases/*"
+  pull_request:
```
Comment (cretz) [inline .github/workflows/build-binaries.yml:7]: Can we remove this?

Comment (dandavison) [reply .github/workflows/build-binaries.yml:7]: Yes, sorry, will do in a subsequent PR since this is almost passing CI.

Comment (dandavison) [reply .github/workflows/build-binaries.yml]: Done

**Review by Sushisource — APPROVED** [review]

**Pull request merged**